src/components/utils/array.js

```diff
@@ -1,2 +1,6 @@
 export const cleanFalsy = list => Array.filter(list, n => n)
+
 export const wrapArray = maybeArray => Array.isArray(maybeArray) ? maybeArray : [maybeArray]
+
+export const flatten = (...values) =>
```

Comment:
is `Array.flat()` part of babel runtime?

Reply:
It doesn't seem to be included, even though were using stage-2 and `Array.flat` has made it to stage-3